packages/material-ui/src/Unstable_TrapFocus/Unstable_TrapFocus.test.js

```diff
@@ -1,7 +1,7 @@
 import * as React from 'react';
 import { useFakeTimers } from 'sinon';
 import { expect } from 'chai';
-import { createClientRender, fireEvent, screen } from 'test/utils';
+import { createClientRender, fireEvent, screen, userEvent } from 'test/utils';
```

Comment:
Could you inline the code in `test/utils` from `user-event` that you used? Last time I checked the `user-event` package had a lot of flaws and simplifications. As a component library we need to be acutely aware of all the events fired.

Reply:
Absolutely.  Are you referring to importing userEvent from @testing-library/user-event rather than from test/utils for this file?

Reply:
Just copy and paste the code into this repository that you need.

Reply:
Which will solve the fail of the CI in the older browsers 👍